src/Address/GeopuntAddressParser.php

```diff
@@ -102,6 +112,11 @@ public function parse(string $formattedAddress): ?ParsedAddress
         }
         $result = $data->LocationResult[0];
 
+        if ($result->Thoroughfarename === null) {
```

Comment:
To me, it seemed more logical to catch this here, than in the `RdfProjector`.

Reply:
I would not make this method return `null` when `Thoroughfarename` is `null`. Maybe other parts of the code need another part, like `Zipcode`, but this no longer works now.

Reply:
I've checked this in the code, and `getPostalCode()` & `getMunicipality()` are used to store it in the cache, but not used anywhere else. Only `getThoroughfare()` & `getHouseNumber()` are used. While I do not think it's likely that GeoPunt would return a `null` on `getThoroughfare()` and an actual value on `getHouseNumber()`, I would consider it incomplete data because a house number without a street isn't really useful information.

Reply:
But the returned object is `ParsedAddress` with all fields `nullable`, what makes it correct or incorrect?

Reply:
Nothing, I doubted between those 2 approaches, but I assumed it would be better to not use a `ParsedAddress` because it is only used for `getThoroughfare()` & `getHouseNumber()`

Reply:
I would keep it broad so the consumer can decide what to do with the result.